azure-governance-cloudwise/ServicePrincipal.ps1

```diff
@@ -0,0 +1,134 @@
+﻿cls
```

Comment:
Seems like this should be part of the template itself?  Any reason why it cannot be?